src/esp/physics/bullet/BulletArticulatedObject.cpp

```diff
@@ -491,6 +491,27 @@ std::vector<float> BulletArticulatedObject::getJointPositions() {
   return positions;
 }
 
+std::vector<float> BulletArticulatedObject::getJointMotorTorques(
+    double fixedTimeStep) {
+  std::vector<float> torques(btMultiBody_->getNumDofs());
+  for (auto& motor : jointMotors_) {
+    btMultibodyLink& btLink = btMultiBody_->getLink(motor.second->index);
+    // int startIndex = velocities ? btLink.m_dofOffset : btLink.m_cfgOffset;
+    auto& settings = motor.second->settings;
+    if (settings.motorType == JointMotorType::SingleDof) {
+      auto& btMotor = articulatedJointMotors.at(motor.first);
+      btScalar impulse = btMotor->getAppliedImpulse(0);
+      btScalar force = impulse / fixedTimeStep;
```

Comment:
May help with clang-tidy issue. Could need some other explicit cast or manipulation.
```suggestion
      double force = impulse / fixedTimeStep;
```